lib/event.rb

```diff
@@ -0,0 +1,56 @@
+class Event
+  attr_reader :name, :food_trucks
+
+  def initialize(name)
+    @name = name
+    @food_trucks = []
+  end
+
+  def add_food_truck(food_truck)
+    @food_trucks << food_truck
+  end
+
+  def food_truck_names
+    @food_trucks.map {|truck| truck.name}
+  end
+
+  def food_trucks_that_sell(item_name)
+    @food_trucks.select {|truck| truck.inventory.include?(item_name)}
+  end
+
+  def sorted_item_list
+    item_names = []
+    @food_trucks.each do |food_truck|
+      inventory = food_truck.inventory
+
+      inventory.each do |item, quantity|
+        false unless item_names.include?(item.name)
+          item_names << item.name
+      end
+    end
+    item_names.sort!.uniq!
```

Comment:
This is fine, but bang operators aren't needed here since you're not doing anything else with the data after this step.